app/components/concerns/permitted_development_rightable.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module PermittedDevelopmentRightable
```

Comment:
For some reason in my head you only do concerns for models. Never thought that you can do the same for the other directories as well.